torch/ao/quantization/experimental/fake_quantize.py

```diff
@@ -0,0 +1,32 @@
+import torch
+from torch.ao.quantization.experimental.observer import APoTObserver
+from torch.ao.quantization.experimental.quantizer import APoTQuantizer, quantize_APoT, dequantize_APoT
+from torch.ao.quantization.fake_quantize import FakeQuantizeBase
+
+class APoTFakeQuantize(FakeQuantizeBase):
+    observer: APoTObserver
+
+    def __init__(self, observer: APoTObserver):
+        super().__init__()
+        self.observer = observer
+
+    def calculate_qparams(self, signed: bool, min_val=None, max_val=None):  # type: ignore[override]
+        return self.observer.calculate_qparams(signed=signed, min_val=min_val, max_val=max_val)
+
+    def forward(self, X: torch.Tensor, signed: bool):  # type: ignore[override]
+        alpha = gamma = quantization_levels = level_indices = None
+
+        if self.observer_enabled[0] == 1:
+            self.observer.forward(X)
+            alpha, gamma, quantization_levels, level_indices = self.observer.calculate_qparams(signed)
```

Comment:
if we guard this with `self.observer_enabled[0] == 1`, we should also store quantization paramters in the fake quantize module I think: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fake_quantize.py#L114-L115, otherwise when we do not enable observer, fake quantize would also fail since these values are initialized to None